### PR TITLE
i18n(ru): Add `cli-reference.mdx` translation 

### PR DESCRIPTION
**Pull request by mearashadowfax**
#### Description

This PR adds the Russian translation of the `cli-reference.mdx` page, addressing missing content identified by the Translation Tracker.

Related:
- #7305

### CODE REVIEW

File: src/content/docs/ru/reference/cli-reference.mdx [hunk 1]
```diff
@@ -0,0 +1,480 @@
...
+
+:::note
+Дополнительный `--` перед любым флагом необходим для того, чтобы `npm` передал ваши флаги команде `astro`.
+:::
+
+### скрипты `package.json`
+
+Вы также можете использовать скрипты в `package.json` для более коротких версий этих команд. Использование скриптов позволяет вам использовать те же команды, с которыми вы можете быть знакомы по другим проектам, например `npm run build`.
+
+Следующие скрипты для наиболее распространенных команд `astro` (`astro dev`, `astro build` и `astro preview`) добавляются автоматически при создании проекта с помощью [инструмента `create astro`] (/ru/install/auto/#1-запустите-мастер-настройки).
```
Comment (yanthomasdev) [inline src/content/docs/ru/reference/cli-reference.mdx:96]:
```suggestion
Следующие скрипты для наиболее распространенных команд `astro` (`astro dev`, `astro build` и `astro preview`) добавляются автоматически при создании проекта с помощью [инструмента `create astro`](/ru/install/auto/#1-запустите-мастер-настройки).
```

**Comment by yanthomasdev** [comment]
Thanks @mearashadowfax, we had a few changes to this page in the latest release, could you take a look and update accordingly? 🙌

**Comment by mearashadowfax** [comment]
@yanthomasdev Certainly, will check it out now. Thank you for fixing the link, again!

**Review by yanthomasdev — APPROVED** [review]
LGTM! Good job 🙌

**Pull request merged**